pkg/lint/lint.go

```diff
@@ -24,6 +24,8 @@ func Run(exe utils.Executor, _ *config.Settings, opts *Options) error {
 		args = append(args, "--image", opts.LinterImage)
 	}
 
+	args = append(args, "-e", "LINTER_RULES_PATH=tmp/") // Prevents mega-linter from spamming lint configuration files into the repository
```

Comment:
Right now it looks like it is using tmp in the current dir, should this be /tmp?

```suggestion
	args = append(args, "-e", "LINTER_RULES_PATH=/tmp") // Prevents mega-linter from spamming lint configuration files into the repository
```

Reply:
Seems to work in both cases. Changed to /tmp as this seems a bit more reliable.